app/controllers/Application.scala

```diff
@@ -761,6 +767,13 @@ object Application extends Controller {
     ).getOrElse(NotFound(views.html.static.trouble("No such subscriber: " + id)))
   }
 
+  def subscriptionBrowse(filter: String, value: Int, page: Int) = Action { implicit request =>
+    filter match {
+      case "scheme" => Ok(views.html.subscription.browse(1, Subscription.inScheme(1, value, page), filter, value, page, Subscription.schemeCount(1, value)))
```

Comment:
I'm trying to envision how this will work when multiple subscribers are enabled... will the method accept a `subscriber` parameter? If so, maybe switching to that and defaulting it to `1` for now would be clearer?